src/main/resources/swagger.api/paths/claimItemReturned/claimItemReturned.yaml

```diff
@@ -0,0 +1,29 @@
+post:
+  operationId: claimItemReturned
+  description: Claims item returned
+  tags:
+    - claimItemReturned
+  parameters:
+    - name: loanId
+      in: path
+      required: true
+      schema:
+        type: string
+        format: uuid
+      description: The ID of the loan
+  requestBody:
+    content:
+      application/json:
+        schema:
+          $ref: '../../schemas/dto/circulation/claimItemReturnedRequest.yaml'
+    required: true
+  responses:
+    '204':
+      description: The loaned item has been successfully claimed returned
+    '422':
+      $ref: '../../responses/unprocessableEntityResponse.yaml'
+    '404':
```

Comment:
Does mod-circulation ever return a 404 from claim returned endpoint?

Reply:
yes, it does: https://s3.amazonaws.com/foliodocs/api/mod-circulation/r/claim-item-returned.html#circulation_loans__id__claim_item_returned_post